src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/TelephonyPolicy.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+ *  Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using System;
+
+namespace Tizen.Security.DevicePolicyManager
+{
+    /// <summary>
+    /// The TelephonyPolicy provides methods to manage telephony policies.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class TelephonyPolicy
+    {
+        private readonly string _messagingPolicyName = "messaging";
+        private readonly DevicePolicyManager _dpm;
+        private int _messagingCallbackId;
+
+        private Interop.DevicePolicyManager.PolicyChangedCallback _messagingPolicyChangedCallback;
+        private EventHandler<PolicyChangedEventArgs> _messagingPolicyChanged;
+
+        internal TelephonyPolicy()
+        {
+        }
+
+        internal TelephonyPolicy(DevicePolicyManager dpm)
+        {
+            _dpm = dpm;
+        }
+
+        /// <summary>
+        /// Checks whether the text messaging is allowed or not.
+        /// </summary>
+        /// <param name="simId">SIM identifier</param>
+        /// <returns>true if the messaging is allowed, false otherwise.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// <exception cref="ArgumentException">Thrown when failed because of invalid parameter.</exception>
+        /// <exception cref="TimeoutException">Thrown when failed because of timeout.</exception>
+        public bool GetMessagingState(string simId)
```

Comment:
How about rename to  `IsMessagingAllowed`
```suggestion
        public bool IsMessagingAllowed(string simId)
```